src/Build.UnitTests/Graph/GraphLoadedFromSolution_tests.cs

```diff
@@ -646,28 +647,6 @@ IEnumerable<ProjectItemInstance> GetIncomingEdgeItemsToNode(ProjectGraphNode nod
             }
         }
 
-        [Fact]
-        public void GraphConstructionShouldThrowOnMissingSolutionDependencies()
```

Comment:
yes, because in the old parser we would read all project dependencies, and if that id was missing in the solution file amongst projects then we throw that exception. but the new parser in this case acts differently - they just don't add the dependency, and this exception is not thrown.
added more details to the PR description about the deleted and moved tests